### ISSUE TITLE
Awesome API! Suggestion for ease of access

### ISSUE THREAD

**Issue by xXGoziXx** [Glowstudent777/YouVersion-API#5]
Hi Glowstudent777,

Fellow Christian developer here. Thanks for making this. Have you considered making this an [npm](https://www.npmjs.com/) package that can be installed? 
There's also a platform called [RapidAPI](https://rapidapi.com/hub) that lets you create an endpoint that people can use.
Just a few suggestions that could make it easier to add this to projects :)

Godstime

**Comment by Glowstudent777** [comment]
Hi, I do actually have a published NPM package.

Here are the links to them:

https://github.com/Glowstudent777/YouVersion-API-NPM

https://www.npmjs.com/package/@glowstudent/youversion

**Comment by xXGoziXx** [comment]
Sweet! I'll have a look at them.

**Comment by xXGoziXx** [comment]
It would be so cool if I could trigger an endpoint that highlighted certain verses. I have so many ideas haha.

**Comment by xXGoziXx** [comment]
Like for example, if a user is viewing a verse online on some website, someone could make an extension that lets users click on the verse which shows a context menu that allows them to highlight the verse in their youversion bible, so they can look more into it later.

**Comment by Haluska77** [comment]
Hey men, fantastic API. I used it for my Homeassistant project, that gives me verse of the day in picture and in word.
I adjusted the url for localize purpose as my native language is slovak. 
Maybe this could be an enhancement for this api.
Thank you so much for that. God bless you.

**Comment by Glowstudent777** [comment]
Hi @Haluska77 ,

I will certainly look into this but from my understanding locales are done through versions. For example I believe NR06 is Italian. Here's an example of the version file that includes localized versions:

```json
{
  "AMP": 1588,
  "ICL00D": 1196,
  "KJV": 1,
  "NIV": 111,
  "NLT": 116,
  "NR06": 122,
  "VULG": 823
}
```

https://github.com/Glowstudent777/YouVersion-API/blob/main/src%2Fapi%2Fv1%2Fdb%2Fversions.json

If this isn't the case please respond with your modifications and I'll look into providing locales correctly.

Best Regards,
Glow

**Comment by Haluska77** [comment]
I simply put location:
https://www.bible.com/sk/verse-of-the-day
Versions you provided are bible versions. But each location has some 'default' bible version. For my purpose and many others might be enough to have at least one local version.

**Comment by Glowstudent777** [comment]
Hi @Haluska77,

Thank you for the info, I'll get started on this soon. Is this the same or similar for verses or just the votd? I'll add SK to the locales and anymore you can think of.

Best Regards,
Glow

**Comment by Haluska77** [comment]
In the following link
https://www.bible.com/sk/bible/1/JHN.1.KJV
There is sk localisation that makes your web in the sk language. But verse is written in the KJV bible version (code 1)

**Comment by Glowstudent777** [comment]
Resolved localization for VOTD in #8 / #9 (fixed tests for the original pr)
